### PR TITLE
nheko: 0.7.2 -> 0.8.0

### PR DESCRIPTION
**Pull request by maralorn**
Includes
mtxclient: 0.3.1 -> 0.4.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/mtxclient/default.nix [hunk 1]
```diff
@@ -14,13 +14,13 @@
 
 stdenv.mkDerivation rec {
   pname = "mtxclient";
-  version = "0.3.1";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/mtxclient/default.nix:17]: Please put this update into a separate commit.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110269` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>mtxclient</li>
<li>nheko</li>
</ul>
</details>

**Comment by pstn** [comment]
This is missing support for the calling feature which needs gstreamer and plugins at build time to be enabled.

**Comment by pstn** [comment]
Also the dependency on libsoduim has not been removed. Please read [release notes](https://github.com/Nheko-Reborn/nheko/releases/tag/v0.8.0) before blindly updating packages.

**Comment by maralorn** [comment]
Hey @pstn thanks for the feedback. I actually read the release notes. (At least most of it.) I just didn‘t know that  "for packagers" notes at the end of release notes are a thing.

**Comment by rnhmjoj** [comment]
I'm working on enabling the voip support. I tried adding gstreamer and the recommended plugins (base, good, bad) but it's still missing something: the embedded video player is not working and call don't start.
